.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        python-version: ['3.6', '3.7', '3.8']
+        python-version: ['3.8']
         toxenv: ['py']
         include:
```

Comment:
Include section won't be needed anymore after adding `quality` in the `toxenv` list.